apps/site/lib/site_web/controllers/schedule/green.ex

```diff
@@ -55,6 +55,13 @@ defmodule SiteWeb.ScheduleController.Green do
     |> render("show.html", [])
   end
 
+  def shuttles(conn, _params) do
+    conn
+    |> assign(:tab, "shuttles")
+    |> put_view(ScheduleView)
+    |> render("show.html", [])
+  end
```

Comment:
Any particular reason we're implementing this page two different ways? For the green line it's a function added to the existing GL schedule controller (here), but for other lines it's a standalone controller.

Reply:
Not especially, except that in doing so I'm replicating the existing pattern. It might be desirable to split up this big controller but that's a job I'm willing to leave for another time.

Reply:
Following the existing pattern seems as good a justification as any, under the circumstances.